pkg/config/config.go

```diff
@@ -93,13 +93,20 @@ type Buildkit struct {
 
 	Secrets map[string]string `json:"secrets" yaml:"secrets,omitempty"`
 
+	Registries map[string]*RegistryConfig `json:"registries,omitempty" yaml:"registries,omitempty"`
```

Comment:
Should this be a pointer? We wouldn't want a a mapping to nil and we don't need to modify it, right?

Reply:
Nope, just instinct. Updated.